### PR TITLE
[fuzz] Added validations for WeightedCluster and AdaptiveConcurrency

### PR DESCRIPTION
**Pull request by zasweq**
Commit Message: Added validations for WeightedCluster and AdaptiveConcurrency
Additional Description:
Risk Level: Low
Testing: N/A
Docs Changes: N/A
Release Notes: N/A

### CODE REVIEW

File: api/envoy/extensions/filters/http/adaptive_concurrency/v3/adaptive_concurrency.proto [hunk 1]
```diff
@@ -54,7 +54,7 @@ message GradientControllerConfig {
     // The time interval between recalculating the minimum request round-trip time.
     google.protobuf.Duration interval = 1 [(validate.rules).duration = {
       required: true
-      gt {}
+      gte {nanos: 1000000}
```
[the remark this answers is not in the document]
Comment (zasweq) [reply api/envoy/extensions/filters/http/adaptive_concurrency/v3/adaptive_concurrency.proto:57]: Do you want me to add a unit test?

[the remark this answers is not in the document]
Comment (zasweq) [reply api/envoy/extensions/filters/http/adaptive_concurrency/v3/adaptive_concurrency.proto:57]: Added comment, but does this Is it easy to add a simple regression test in AdaptiveConcurrencyFilterTest that would have crashed before? Mean to add one?

[the remark this answers is not in the document]
Comment (asraa) [reply api/envoy/extensions/filters/http/adaptive_concurrency/v3/adaptive_concurrency.proto:58]: If it's easy! I think it should be. There looks to be existing yaml tests when I checked.

**Review by asraa — APPROVED** [review]
Thanks for the tests! LGTM

**Review by mattklein123 — APPROVED** [review]
Thanks!

**Pull request merged**